tools/reviewer/local_server/service/code_review.proto

```diff
@@ -53,6 +53,17 @@ message PongResponse { string message = 1; }
 
 message RemoveWorkspaceRequest { string workspace_name = 1; }
 
+message GithubRepo {
+  string owner = 1;
+  string repo = 2;
+}
+
+message GithubPr {
+  GithubRepo repo = 1;
+  // PR number - essentially an ID
```

Comment:
This ID is set by GitHub right? Please add that to the comment.

Reply:
Yes, it is set by GitHub.
Done.